packages/create-cloudflare/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # create-cloudflare
 
+## 2.26.0
+
+### Minor Changes
+
+- [#6573](https://github.com/cloudflare/workers-sdk/pull/6573) [`3ff6506`](https://github.com/cloudflare/workers-sdk/commit/3ff6506cc5f1d79e5df6530673cb600507c7da1b) Thanks [@penalosa](https://github.com/penalosa)! - feature: Use `create-vite` instead of `create-react-app` for React projects.
```

Comment:
As mentioned in the PR, credit is to be given to the original author https://github.com/cloudflare/workers-sdk/pull/6573#issuecomment-2313227028

```suggestion
- [#6573](https://github.com/cloudflare/workers-sdk/pull/6573) [`3ff6506`](https://github.com/cloudflare/workers-sdk/commit/3ff6506cc5f1d79e5df6530673cb600507c7da1b) Thanks [@santraez](https://github.com/penalosa)! - feature: Use `create-vite` instead of `create-react-app` for React projects.
```